h2o-r/h2o-package/R/parse.R

```diff
@@ -67,6 +70,9 @@ h2o.parseRaw <- function(data, pattern="", destination_frame = "", header=NA, se
             decrypt_tool = .decrypt_tool_id(parse.params$decrypt_tool),
             skipped_columns = paste0("[", paste(parse.params$skipped_columns, collapse=','), "]")
             )
+   if(!is.null(custom_non_data_line_markers)){
+    parse.params <- append(parse.params,list(custom_non_data_line_markers = custom_non_data_line_markers))
```

Comment:
whitespace, we use 2 space indent

Reply:
This place has +2 spaces indentation. I see no problem here.

Reply:
Re-formatted in R studio. For some reason. GitHub won't show the change 

![Snímek z 2019-03-27 14-35-41](https://user-images.githubusercontent.com/8769110/55080013-a7fd6580-509d-
11e9-8920-9060903378a7.png)

Reply:
I have no idea why the change is not shown. It has proper indentation after the re-format. Everything else is ok. Everything is pushed.

![Uploading Snímek z 2019-03-27 14-35-41.png…]()

I even broken it intentionally in https://github.com/h2oai/h2o-3/pull/3380/commits/82c557a520572cbc98a0e238aeb7fc06cb627f25 and then fixed it back in https://github.com/h2oai/h2o-3/pull/3380/commits/cd6352c108608c7c5a9ea48f32106709612fba54 - no luck.

Reply:
Great, now it is shown correctly even here, not sure what was wrong with github.

Reply:
Here, it looks like `  if(!is.null(custom_non_data_line_markers)){` has three spaces before it, in reality it only has two. See the commit links ^^^